### PR TITLE
Fix/profile UI

### PR DESCRIPTION
**Pull request by OkoloJohnbosco**
# Fixes Issue/Linear Ticket
Fixed profile UI section

# Changes proposed

1. Profile should be aligned with the top navigation.
2. On view profile click profile modal should close.

## What did you do?

1. Aligned profile section with the top navigation.
2. Edit status now pops up an emoji picker on an overlay backdrop
3.  Added the click outside hook.

# Check List (Check all the applicable boxes)

🚨Please review the [style guide for contributing](https://github.com/zurichat/zc_main/blob/dev/docs/STYLING.md) and [guidelines for contributing](https://github.com/zurichat/zc_main/blob/dev/docs/CONTRIBUTING.md) to this repository.

- [X] My code follows the code style of this project.
- [X] This PR does not contain plagiarized content.
- [X] The title and description of the PR is clear and explains the approach.
- [X] I am making a pull request against the **dev branch** (left side).
- [X] My commit messages styles matches our requested structure.
- [X] My code additions will fail neither code linting checks nor unit test.
- [X] I am only making changes to files I was requested to.

# Screenshots/ Videos
[New Recording - 27_11_2022, 16_40_37.webm](https://user-images.githubusercontent.com/47562075/204144024-1043ca48-37cf-41e4-a3e6-937d663eb1e6.webm)

### CODE REVIEW

File: packages/main/src/components/protected/topbar/components/NotificationPreference.jsx [hunk 1]
```diff
@@ -12,6 +12,7 @@ const NotificationPreference = () => {
   const [notificationSettings, setNotificationSettings] = useState(
     user.settings?.notifications
   );
+  console.log(user.settings?.notifications);
```
Comment (billmal071) [inline packages/main/src/components/protected/topbar/components/NotificationPreference.jsx:19]: Remove.

File: packages/main/src/components/protected/topbar/components/NotificationPreference.jsx [hunk 2]
```diff
@@ -33,6 +34,24 @@ const NotificationPreference = () => {
     { name: "after i've been inactive for 30 minute" }
   ]);
   const [notificationSend, setNotificationSend] = useState("");
+
+  const getData = () => {
+    authAxios
+      .get(`/organizations/${user.org_id}/members/${user._id}`)
+      .then(res => {
+        console.log("get data res =>", res.data);
```
Comment (billmal071) [inline packages/main/src/components/protected/topbar/components/NotificationPreference.jsx:46]: Remove all your console.log

**Review by billmal071 — CHANGES_REQUESTED** [review]

File: packages/main/src/components/protected/topbar/components/NotificationPreference.jsx [hunk 3]
```diff
@@ -16,6 +16,7 @@ const NotificationPreference = () => {
   const [notificationSettings, setNotificationSettings] = useState(
     user.settings?.notifications
   );
+  console.log(user.settings?.notifications);
```
Comment (JulianaSau) [inline packages/main/src/components/protected/topbar/components/NotificationPreference.jsx:19]: Remove this console.log

**Review by JulianaSau — CHANGES_REQUESTED** [review]

File: packages/main/src/components/protected/topbar/components/NotificationPreference.jsx (hunk 1 above)
Comment (OkoloJohnbosco) [reply packages/main/src/components/protected/topbar/components/NotificationPreference.jsx:19]: Ok boss

**Review by JulianaSau — APPROVED** [review]

**Pull request merged**